_includes/releases/cloud/2023-05-15.md

```diff
@@ -0,0 +1,16 @@
+## May 15, 2023
+
+In addition to many of the Feature Highlights in the [CockroachDB v23.1.0 Release Notes](v23.1.html), the following features are specifically available to Cloud clusters once they are upgraded:
```

Comment:
suggest making this link to feature highlights a direct link: https://www.cockroachlabs.com/docs/releases/v23.1.html#v23-1-0-feature-highlights